_src/docs.css

```diff
@@ -478,6 +548,155 @@ hr {
   font-display: swap;
 }
 
+/** Custom Chain Cards **/
```

Comment:
If this CSS is specific to only the home page, can we scope it to that page only (move the CSS to that page?)

Reply:
Will do this in a followup.